examples/python/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##
+##                          _                      
+##                         | |                     
+##                       __| | __ ___      ___ ___  
+##                      / _` |/ _` \ \ /\ / / '_  | 
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+# Protobuf_DIR is <protobuf_root>/lib/cmake/protobuf
+get_filename_component(camke_dir  "${Protobuf_DIR}" DIRECTORY)
+get_filename_component(lib_dir "${camke_dir}" DIRECTORY)
+get_filename_component(protobuf_root "${lib_dir}" DIRECTORY)
+
+#
+# Check the python libraries were installed in <protobuf_root>/python/
+set(protobuf_python "${protobuf_root}/python")
+set(protobuf_init_py "${protobuf_python}/google/protobuf/__init__.py")
+
+if(NOT EXISTS "${protobuf_init_py}")
+  message(WARNING 
+    "Python module of protobuf is not installed correctly! Expected protobuf's __init__.py at:\n"
+    "    ${protobuf_init_py}\n"
+    "Disabling python module of Dawn."
+  )
+else()
+  set(DAWN_INSTALL_PROTOBUF_MODULE "${protobuf_python}")
+
+  set(config_in "${CMAKE_SOURCE_DIR}/examples/python/config.py.in")
+  set(config_out "${CMAKE_BINARY_DIR}/examples/python/config.py")
```

Comment:
should the config be written to `${CMAKE_BINARY_DIR}/examples/python/config.py` and not to `${CMAKE_SOURCE_DIR}/examples/python/config.py`? I would expect to be able to run the examples built in the example directory and not in the install

Reply:
I think in general is good practice to not touch the original source code during the build. 
The config also sets some directories to add to the python path, therefore it would be complicated to allow to build the module in multiple paths, so I would advise against

Reply:
ok, I agree